src/sinks/influxdb/metrics.rs

```diff
@@ -796,6 +820,42 @@ mod tests {
         );
         assert_eq!("1542182950000000011", line_protocol.3);
     }
+
+    #[test]
+    fn test_encode_with_some_tags() {
```

Comment:
Can you add `crate::test_util::trace_init()` here? :)

Reply:
Should I add it to every test also ?

Reply:
Just the ones you add is fine. 😓